tutorial/README.md

```diff
@@ -0,0 +1,120 @@
+# Data Preparation
+
+Viv supports a subset of the files generated from the `bioformats2raw` + `raw2ometiff` pipeline, described 
+[here](https://www.glencoesoftware.com/blog/2019/12/09/converting-whole-slide-images-to-OME-TIFF.html). 
+This guide demonstrates how to generate a pyramidal Zarr or OME-TIFF with Bio-Formats that can be viewed with 
+[Avivator](http://avivator.gehlenborglab.org) via HTTP.
+
+
+### Getting Started
+
+This tutorial requires Bio-Formats [`bioformats2raw`](https://github.com/glencoesoftware/bioformats2raw) and 
+[`raw2ometiff`](https://github.com/glencoesoftware/raw2ometiff) command-line tools. It's easiest to install 
+these tools using [`conda`](https://docs.conda.io/projects/conda/en/latest/user-guide/install/), but binaries also are available for download on the corresponding github repositories.
+
+```bash
+$ conda create --name bioformats python=3.8
+$ conda activate bioformats
+$ conda install -c ome bioformats2raw raw2ometiff
+```
+
+
+### Input data
+
+Bio-Formats is an incredibly valuable toolkit and supports reading over 
+[150 file formats](https://docs.openmicroscopy.org/bio-formats/6.5.1/supported-formats.html). You can choose 
+one of your own images, but for the purposes of this tutorial, we will use a multiplexed, high-resolution 
+[Perkin Elmer](https://downloads.openmicroscopy.org/images/Vectra-QPTIFF/perkinelmer/) (1.95 GB) image made available 
+under [CC-BY 4.0](https://creativecommons.org/licenses/by/4.0/) on OME.
+
+```bash
+$ wget https://downloads.openmicroscopy.org/images/Vectra-QPTIFF/perkinelmer/PKI_scans/LuCa-7color_Scan1.qptiff 
+```
+
+After the image has finished downloading, there are two options for creating an Avivator/Viv-compliant image. 
+
+
+### Pyramid Generation
+
+#### Option 1: Create a Bio-Formats "raw" Zarr
+
+The first option is to use `bioformats2raw` with `--file_type=zarr`. The default "raw" file type is currently 
+[n5](https://github.com/saalfeldlab/n5), so the flag is required to generate the zarr-based output. This command will
+create the OME-XML metadata along with a pyramidal Zarr for high-resolution images.
+
+```bash
+$ bioformats2raw LuCa-7color_Scan1.qptiff LuCa-7color_Scan1/ --file_type=zarr
+```
+
+`bioformats2raw` creates the file directory `LuCa-7color_Scan1/` which contains the "raw" bioformats output. The root directory 
+contains a `METADATA.ome.xml` file along with a `data.zarr/` directory containing the zarr
+output. This output can be viewed directly with [Avivator] by serving the top-level directory (`LuCa-7color_Scan1/`) 
+over HTTP ([see below](#viewing-in-avivator)).
+
+
+#### Option 2: Create an OME-TIFF
+
+The second option is to run the complete Bio-Formats pipeline to generate a valid OME-TIFF. 
+
+```bash
+$ bioformats2raw LuCa-7color_Scan1.qptiff n5_tile_directory/ 
```

Comment:
```suggestion
$ bioformats2raw LuCa-7color_Scan1.qptiff n5_tile_directory/ --tile_width 512 --tile_height 512
```

I think we should use 512 tile sizes.  This is what Mapbox does, I think, and it's what we do in the portal. I think it's a nice balance between screen sizes and data loading.

Reply:
I think this a good suggestion, but I want the tutorial to feel very simple. Explicit tile sizes are potentially an optimization, but we haven't actually tested for comparison. 

The default tilesize is `1024` and this is how the IDR is producing their pyramidal data. I'd hesitate to suggest something else unless we have a really convincing reason. We could add a footnote that other parameters (like tilesize) can be specified for `bioformats2raw`, but if `tile_width` and `tile_height` are different viv won't work. Thoughts, @ngehlenborg?

Ideally I'd like the tutorial to be run `bioformats2raw` and `raw2ometiff` without any flags and it just works, but that's not the case.

Reply:
How about suggesting image tile sizes with a link to the bioformats2raw docs that explain how to set something different from the default?

Reply:
I'm just hesitant to make a "suggested" tile size since we don't really know. Larger tiles means fewer requests, but maybe larger unnecessary data transfer. It's hard to know.  

I'd rather say "You can adjust tile sizes if desired by ____", but my understanding is the tile size of `1024` is chosen currently because the number of files per directory is enormous for the base image, making it difficult to work with that many objects on local disk. By changing to 512, you increase the number of files per directory by 4.

Reply:
I just added a note. Please let me know if more is necessary.

Reply:
We would suggest what has well worked for us (512, 1024). Otherwise people might choose tile sizes that don't work (well) and if we don't offer any guidance, they will just be frustrated and decide that Viv doesn't work for them.

Reply:
Added. Thanks.